blocks/leadspace/leadspace.css

```diff
@@ -1,4 +1,36 @@
 /* Leadspace block styles */
+.leadspace, .leadspace p {
+    font-family: var(--sans-serif-font-light);
+    font-size: var(--font-size-16);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-160);
+    letter-spacing: var(--letter-spacing-1);
+    }
+
+.leadspace h1, h2 {
```

Comment:
`.leadspace h2`?

Reply:
Yes, per Amol and my convo, we can see from the Figma designs that the customer fully expects to be able to change the headline to either size at will. So I needed to include both here, despite semantics' best practices...

Reply:
But shouldn't it be `.leadspace h1, .leadspace h2`?

Reply:
OH, LOL! Yes, yes it should!!